docs/source/requests.md

```diff
@@ -1469,6 +1475,192 @@ Let's consider an example of changing a value of a NODE transaction's `service`
 ```
 
 
+### AUTH_RULES
+
+A command to set multiple AUTH_RULEs by one transaction. 
+Transaction AUTH_RULES is not divided into two AUTH_RULE transaction, and is written to the ledger with one transaction with the full set of rules that come in the request.
+Internally authentication rules are stored as a key-value dictionary: `{action} -> {auth_constraint}`.
+
+The list of actions is static and can be found in [auth_rules.md](auth_rules.md).
+There is a default Auth Constraint for every action (defined in [auth_rules.md](auth_rules.md)). 
+
+The `AUTH_RULES` command allows to change the Auth Constraints.
+So, it's not possible to register new actions by this command. But it's possible to override authentication constraints (values) for a given action.
+
+Please note, that list elements of `GET_AUTH_RULE` output can be used as an input (with a required changes) for the field `rules` in `AUTH_RULES`.
+
+If one rule is incorrect, the client will receive NACK message for the request with all its rules.
+A client will receive NACK for 
+- a request with incorrect format;
+- a request with a rule with "ADD" action, but with "old_value";
+- a request with a rule with "EDIT" action without "old_value";
+- a request with a rule with a key that is not in the [auth_rule](auth_rule.md).
+
+The following input parameters must match an auth rule from the [auth_rules.md](auth_rules.md):
+- `auth_type` (string enum)
+ 
+     The type of transaction to change the auth constraints to. (Example: "0", "1", ...). See transactions description to find the txn type enum value.
+
+- `auth_action` (enum: `ADD` or `EDIT`)
```

Comment:
Please add description of `rules` parameter as the top level one. All other params should be sub-items for `rules`.
The same for transactions.md.